faker/providers/person/vi_VN/__init__.py

```diff
@@ -0,0 +1,1670 @@
+# coding=utf-8
+
+from __future__ import unicode_literals
+from collections import OrderedDict
+
+from .. import Provider as PersonProvider
+
+
+# I statstic with data from https://github.com/duyetdev/vietnamese-namedb
```

Comment:
You can just include the URL, no need for words :)

Reply:
this is a repository for nature language processing, i believe it

Reply:
What I"m saying is to replace `I statstic with data from https://github.com/duyetdev/vietnamese-namedb` with just `Data from https://github.com/duyetdev/vietnamese-namedb`